docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml

```diff
@@ -592,30 +600,190 @@ depending on where they wind up.-->
         <para>The package maps page, displays all the package maps loaded on
         your cluster. You can Add, Activate, Deactivate, Delete, or Open a
         package map. To examine a package map, select a package map from the
-        list.</para>
+        list. </para>
 
-        <para>You can select the <emphasis role="bold">Validate Package
-        Map</emphasis> tab to validate a package map.</para>
+        <para>To update the package maps you are using, you would either edit
+        the package map file (or copy then add) or add a new one. Then
+        activate the new one, and then delete the old one. </para>
```

Comment:
To update the package maps you are using, you would either edit the package map file or add a new one and then activate it. You could later delete the old one.